applications/Unity.GrantManager/src/Unity.GrantManager.HttpApi/Controllers/IntakeSubmissionController.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Unity.GrantManager.Models;
+using Volo.Abp.AspNetCore.Mvc;
+using Volo.Abp.Domain.Repositories;
+
+namespace Unity.GrantManager.Controllers
+{
+    [ApiController]
```

Comment:
I'm assuming the separate controller here as apposed to using the Service Layer as we will use this as a 'B2B' integration and can then lock this down specifically away from the application tier?